unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/WorldBlockerHandler.cs

```diff
@@ -0,0 +1,98 @@
+﻿using DCL.Configuration;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Controllers
+{
+    public class WorldBlockerHandler : BlockerHandler
```

Comment:
Why do we need the `WorldBlockerHandler` and `BlockerHandler`? Why don't use just a single class?

Reply:
Removed old blockers code and transformed the BlockerHandler with the new world blockers logic.